server/config/tagType/tagConfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "REJECTED_BY_PSU": [
```

Comment:
Is an array the correct thing here? Looks to me like theres only one item so can be an object?

Reply:
Agreed and will incorporate this.